blog_on_rails/db/seeds.rb

```diff
@@ -0,0 +1,29 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+Post.destroy_all()
+Comment.destroy_all()
+
+100.times do
+    p = Post.create(
+        title: Faker::TvShows::Buffy.episode,
+        body: Faker::Lorem.paragraph(sentence_count: 5, supplemental: true, random_sentences_to_add: 4),
+        created_at: Faker::Date.backward(days:365 * 5),
+        updated_at: Faker::Date.backward(days:365 * 5)
```

Comment:
To prevent the `updated_at` date from being older than the `created_at` date, a `Faker::Date` can be saved in a variable such as `past_date` in each iteration of the loop like so: `past_date = Faker::Date.backward(days:365 * 5)`, and the same date can be used for both dates:

```ruby
        created_at: past_date,
        updated_at: past_date
```